docs/.vitepress/theme/custom.css

```diff
@@ -0,0 +1,6 @@
+:root {
+    --vp-c-brand-1: #AA0000;
+    --vp-c-brand-2: #a22a21;
+    --vp-c-brand-3: #AA0000;
+    --vp-c-brand-soft: #AA0000;
+}
```

Comment:
trailing newlineeeee

Reply:
Added trailing newlineeeeee in 1289ab2e23a0b185962533363eaa925ef6f50cb1